tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/NewLiferayModuleProjectOpMethods.java

```diff
@@ -48,8 +49,8 @@ public static final Status execute( final NewLiferayModuleProjectOp op, final Pr
             final NewLiferayProjectProvider<NewLiferayModuleProjectOp> projectProvider = op.getProjectProvider().content( true );
 
             //IDE-1306  If the user types too quickly all the model changes may not have propagated
-            final Path projectLocation = op.getLocation().content();
-            updateLocation( op, projectLocation );
+            //final Path projectLocation = op.getLocation().content();
```

Comment:
Before our ide will invoke the build script to create project, we need to append project name in location path, but I found the blade script didn't need to know this and we just need to set project  parent folder location. So I think we can remove it.